packages/vue/src/components/FContextMenu/FContextMenu.vue

```diff
@@ -1,212 +1,201 @@
+     * after pressing for example esc in the menu
+     */
+    close: [];
+    /**
+     * Event that is dispatched when an item is selected.
+     * @type {string} item key
+     */
+    select: [key: string];
+}>();
+
+const $t = useTranslate();
+
+const preventKeys = ["Tab", "Up", "Down", "ArrowUp", "ArrowDown", "Home", "End", " ", "Spacebar", "Enter", "Escape"];
 const keyUp = ["ArrowUp", "Up"];
 
-export default defineComponent({
-    name: "FContextMenu",
-    components: { IPopup, FIcon },
-    props: {
-        /**
-         * Toggle open/closed popup.
-         */
-        isOpen: {
-            type: Boolean,
-            required: true,
-        },
-        /**
-         * DOM element to position popup at.
-         */
-        anchor: {
-            type: HTMLElement as PropType<HTMLElement | undefined>,
-            required: false,
-            default: undefined,
-        },
-        /**
-         * The items to be displayed in the menu.
-         */
-        items: {
-            type: Array as PropType<ContextMenuItem[]>,
-            required: true,
-        },
-        /**
-         * Unique accessible name for navigation landmark.
-         */
-        ariaLabel: {
-            type: String,
-            required: false,
-            default: "Kontextuell meny",
-        },
-    },
-    emits: [
-        /**
-         * Event that is dispatched after an item is selected or
-         * after pressing for example esc in the menu
-         */
-        "close",
-        /**
-         * Event that is dispatched when an item is selected.
-         * @type {string} item key
-         */
-        "select",
-    ],
-    setup() {
-        return { contextmenu: ref<HTMLElement | null>(null) };
-    },
-    data() {
-        return {
-            selectedItem: "",
-            currentFocusedItemIndex: -1,
-        };
-    },
-    computed: {
-        popupItems(): ContextMenuTextItem[] {
-            return this.items.filter(isContextMenuTextItem);
-        },
-        separatorPositions(): number[] {
-            const res: number[] = [];
-            if (this.items.length > 1) {
-                this.items.forEach((it, i) => {
-                    if (isContextMenuSeparatorItem(it)) {
-                        const pos = i - 1 - res.length;
-                        if (pos >= 0 && pos < this.items.length - 1) {
-                            res.push(pos);
-                        }
-                    }
-                });
-            }
-            return res;
-        },
-        hasIcons(): boolean {
-            return this.items.some((it) => isContextMenuTextItem(it) && it.icon);
-        },
-    },
-    watch: {
-        isOpen: {
-            immediate: true,
-            handler() {
-                if (this.isOpen) {
-                    this.currentFocusedItemIndex = -1;
-                    this.selectedItem = "";
+const contextmenu = useTemplateRef("contextmenu");
```

Comment:
Inget tvunget, men visst landade vi att köra med `Ref`-suffix på template referenser?

```suggestion
const contextmenuRef = useTemplateRef("contextmenu");
```